internal/gc.h

```diff
@@ -73,6 +73,11 @@ struct rb_objspace; /* in vm_core.h */
 
 #define RVARGC_PAYLOAD_INIT(obj, size) (void *)rb_rvargc_payload_init((VALUE)obj, (size_t)size)
 
+typedef struct ractor_newobj_cache {
+    struct RVALUE *freelist;
+    struct heap_page *using_page;
+} rb_ractor_newobj_cache;
```

Comment:
nit: typedefs tend to end with `_t`.
```suggestion
} rb_ractor_newobj_cache_t;
```

Reply:
Thanks for catching that!